File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        sd,st= {},{}
        for i in s:
            if i in sd:
                sd[i]+= 1

            else:
                sd[i] = 1

        for i in t:
            if i in st:
                st[i] += 1
            else :
                st[i] = 1

        ans = 0
        for i in st:
            if i in sd:
                if st[i] > sd[i]:
                    ans += (st[i]-sd[i])

            else:
                ans += st[i]

        return ans

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int x[]= new int[26];
        int y[]= new int[26];
        int count =0;
        for(int i=0;i<s.length();i++)
        {
            x[s.charAt(i)-'a']++;
        }
        for(int i=0;i<t.length();i++)
        {
            y[t.charAt(i)-'a']++;
        }
        for(int i=0;i<26;i++)
        {
            if(x[i]!=y[i] && x[i]>y[i])
            count+=x[i]-y[i];

           // System.out.println("x==>"+x[i]+" y==>"+y[i]);
        }
        return count;
    }
}